src/frontend/src/pages/FinancePage/FinanceComponents/ReportRefundModal.tsx

```diff
@@ -69,9 +75,14 @@ const ReportRefundModal: React.FC<ReportRefundProps> = ({ modalShow, handleClose
           <ReactHookTextField
             name="refundAmount"
             control={control}
-            sx={{ width: 1 }}
-            type="number"
+            sx={{
```

Comment:
what does this do

Reply:
this gets rid of that little button with two arrows on the right side that shows up when you set an input as a number. (was annoying me). I guess we don't need it here now bc I marked this input as not a # (it wouldn't let you submit #'s with decimals when the type was number), but we could use it elsewhere

Reply:
ah i see